frontend/src/concepts/dashboard/codeEditor/DashboardCodeEditor.tsx

```diff
@@ -11,7 +11,7 @@ const DashboardCodeEditor: React.FC<Partial<DashboardCodeEditorProps>> = ({
   height = 'calc(100% - 38px)',
   ...props
 }) => (
-  <div style={{ height }}>
+  <div data-testid="dashboard-code-editor" style={{ height }}>
```

Comment:
Generic components should allow setting of the `data-testid` as an incoming prop. While it may be unlikely that we have multiple code editors on the same page, it's better to follow a consistent pattern for all generic components.